File: src/nomanssave/al.java
package nomanssave;

import javax.swing.ComboBoxModel;
import javax.swing.event.ListDataListener;

class al implements ComboBoxModel {
   String ch;
   // $FF: synthetic field
   final aj cg;

   al(aj var1) {
      this.cg = var1;
      this.ch = null;
   }

   public int getSize() {
      return aj.Q().size();
   }

   public String s(int var1) {
      return (String)aj.Q().get(var1);
   }

   public void addListDataListener(ListDataListener var1) {
   }

   public void removeListDataListener(ListDataListener var1) {
   }

   public void setSelectedItem(Object var1) {
      this.ch = (String)var1;
   }

   public Object getSelectedItem() {
      return this.ch;
   }

   // $FF: synthetic method
   public Object getElementAt(int var1) {
      return this.s(var1);
   }
}

File: NMSSaveEditor/nomanssave/lower/al.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{

public class al : object {
   public string ch;
   aj cg;

   public al(aj var1) {
      this.cg = var1;
      this.ch = null;
   }

   public int getSize() {
      return aj.Q().Count;
   }

   public string s(int var1) {
      return (string)aj.Q()[var1];
   }

   public void addListDataListener(EventHandler var1) {
   }

   public void removeListDataListener(EventHandler var1) {
   }

   public void setSelectedItem(object var1) {
      this.ch = (string)var1;
   }

   public object getSelectedItem() {
      return this.ch;
   }
   public object getElementAt(int var1) {
      return this.s(var1);
   }
}

}
